pre-docs/bolt_command_reference.md

```diff
@@ -1,109 +1,498 @@
-# Bolt command reference
+# Bolt Command Reference
```

Comment:
Puppet corp style guide recommends sentence case for headers (minus any punctuation at the end). So the original version of this is preferred.